config/webpack.js

```diff
@@ -56,18 +50,10 @@ module.exports = (env) => {
       chunkFilename: isDev ? '[name]-[hash].js' : '[name]-[contenthash:6].js'
     },
 
-    // Dev server
-    serve: process.env.WEBPACK_SERVE
-      ? {
-          devMiddleware: {
-            stats: 'errors-only'
```

Comment:
Can you make sure we still have the minimal stats printing out with the dev server? It shouldn't print out a bunch of info about the compiled modules.

Reply:
Done!